src/services/geonode.js

```diff
@@ -1,7 +1,7 @@
 import MapConfigTransformService from 'boundless-sdk/services/MapConfigTransformService';
 import MapConfigService from 'boundless-sdk/services/MapConfigService';
 import {getCRSFToken, removeTrailingSlash} from '../helper';
-const NEW_MAP_ENDPOINT = '/maps/new/data';
+import {edit_map_endpoint, NEW_MAP_ENDPOINT} from '../constants/server'
```

Comment:
yes, not sure if constant is the right place for it. probably not. do you have a good suggestion?